src/app/legacy/psammead/psammead-navigation/src/__snapshots__/index.test.jsx.snap

```diff
@@ -118,6 +118,7 @@ exports[`Navigation should render correctly 1`] = `
   right: 0;
   bottom: 0;
   border-bottom: 0.25rem solid #B80000;
+  border-bottom: 0.3125rem solid #B80000;
```

Comment:
Still trying to work out what's causing these duplicate border-bottom declarations